go.mod

```diff
@@ -43,6 +43,7 @@ require (
 	k8s.io/client-go v8.0.0+incompatible
 	k8s.io/code-generator v0.0.0
 	k8s.io/component-base v0.0.0
+	k8s.io/klog v0.4.0
```

Comment:
OLM codepaths using `glog` were brought into the packageserver dependency tree transitively. This was causing flag registration conflicts with `klog`, so I swapped  all references to `glog` for `klog`.